runhouse/servers/cluster_servlet.py

```diff
@@ -114,6 +119,13 @@ async def aset_cluster_config_value(self, key: str, value: Any):
 
         return self.cluster_config
 
+    async def asave_cluster_config_locally(self, config: Dict[str, Any]):
```

Comment:
why do we need this instead of just updating the cluster config stored in the `ClusterServlet`

Reply:
Yeah, since it is only for internal usage, there is not really a necessity to update the cluster config locally.

Having said that, I do wonder - don't we want the user to be aware that the observability is disabled or enabled on a specific cluster? If such field will be saved on the cluster_config, it could be displayed on cluster metadata in den. 

Another way to display it will be by calling cluster.status(). The cluster_config retrieved by the status method is the cluster_config saved in the cluster_servlet, so we'll be able to know if the cluster telemetry is on of not. We could add Matt to the discussion on how to display it in the UI.

wdyt?